2009-10-05  Andreas Schwab  <schwab@redhat.com>

	* locale/C-time.c: Revert week-1stday back to 19971130 and set
	first_weekday to 1 and first_workday to 2.

diff --git a/locale/C-time.c b/locale/C-time.c
index a9275d9..368ffce 100644
--- a/locale/C-time.c
+++ b/locale/C-time.c
@@ -134,10 +134,10 @@ const struct locale_data _nl_C_LC_TIME attribute_hidden =
     { .wstr = (const uint32_t *) L"" },
     { .wstr = (const uint32_t *) L"" },
     { .string = "\7" },
-    { .word = 19971201 },
+    { .word = 19971130 },
     { .string = "\4" },
-    { .string = "\7" },
     { .string = "\1" },
+    { .string = "\2" },
     { .string = "\1" },
     { .string = "" },
     { .string = "%a %b %e %H:%M:%S %Z %Y" },
